apps/expo-go/android/expoview/src/main/java/host/exp/exponent/kernel/Kernel.kt

```diff
@@ -614,15 +620,12 @@ class Kernel : KernelInterface() {
         manifestUrl,
         object : AppLoaderCallback {
           override fun onOptimisticManifest(optimisticManifest: Manifest) {
-            kernelScope.launch {
-              sendOptimisticManifestToExperienceActivity(
-                optimisticManifest
-              )
-            }
+            Exponent.instance
+              .runOnUiThread { sendOptimisticManifestToExperienceActivity(optimisticManifest) }
```

Comment:
is that intentional to rollback `Exponent.instance.runOnUiThread`? i was thinking that you try to remove this singleton by kernelScope?

Reply:
I was just debugging something and forgot to revert this 😅